src/Pipl.php

```diff
@@ -1,44 +1,42 @@
 <?php
 
 namespace Abdelrahman_badr\Pipl;
+
 use Exception;
-class Pipl 
+
+class Pipl
 {
+    public $key;
 
-    public $key ;
-    public $base_url ;
     public function __construct()
     {
         $this->key = env('PIPL_API_KEY');
-        $this->url = "http://api.pipl.com/search/?&key=".$this->key;
+        $this->url = (env('PIPL_API_BASE_URL') || "http://api.pipl.com/search/") . "?key={$this->key}";
```

Comment:
It would be better if you used the default url in env() function 
`env('PIPL_API_BASE_URL', 'http://api.pipl.com/search/')`

Reply:
You're absolutely right. Completely forgot about that one